packages/pharos/src/components/link/pharos-link.ts

```diff
@@ -57,7 +57,8 @@ export class PharosLink extends FocusMixin(AnchorElement) {
   public bold = false;
 
   /**
-   * Indicates if the link should display as a flex container.
+   * Indicates if the link should display as a flex container. Enable this for a link heading so that
+   * the blue outline is rendered on focus for Chrome and Safari (works as expected on Firefox).
```

Comment:
Is there broader guidance that can be given here? I suspect headings might not be the only thing affected, or if they are someone might need to understand why here.